tests/vmi_multus_test.go

```diff
@@ -578,27 +588,27 @@ var _ = Describe("SRIOV", func() {
 		sriovResourceName = "openshift.io/sriov_net"
 	}
 
-	tests.BeforeAll(func() {
-		tests.BeforeTestCleanup()
-		// Create two sriov networks referring to the same resource name
+	createNetwork := func(networkName string, nameSpace string, networkAttachmentDefinition string) {
 		result := virtClient.RestClient().
 			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov")).
-			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
-		result = virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov2")).
-			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov2", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
-		result = virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov-link-enabled")).
-			Body([]byte(fmt.Sprintf(sriovLinkEnableConfCRD, "sriov-link-enabled", tests.NamespaceTestDefault, sriovResourceName))).
+			RequestURI(fmt.Sprintf(postUrl, nameSpace, networkName)).
+			Body([]byte(fmt.Sprintf(networkAttachmentDefinition, networkName, nameSpace, sriovResourceName))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
+	}
+
+	deleteNetwork := func(networkName string) {
+		Expect(virtClient.RestClient().Delete().Namespace(tests.NamespaceTestDefault).Resource("net-attach-def").Name(networkName).Do().Error()).To(Succeed(),
+			"failed to delete network attachment definition")
+	}
+
+	tests.BeforeAll(func() {
+		tests.BeforeTestCleanup()
+		// Create two sriov networks referring to the same resource name
```

Comment:
i @maiqueb 
It was remained form the previous tests. It was right before creating the two networks, as it is now.
where do think I should move it? or maybe you think it is redundant?

Reply:
I would remove it, since I don't understand it's context now - you're creating 3 different networks... With different resource names. (2 in one namespaces, then 1 in a separate namespace).

That comment just confuses me.

Reply:
Done.